libs/cli/src/test/java/org/opensearch/cli/KeystoreConstantsTests.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.cli;
+
+import org.opensearch.test.OpenSearchTestCase;
+
+public class KeystoreConstantsTests extends OpenSearchTestCase {
```

Comment:
This tests is not very useful to be fair

Reply:
Initially I just added as placeholder but since with the move of `MAX_PASSPHRASE_LENGTH ` to `EnvironmentAwareCommand` removed this.